### ISSUE TITLE
Generate GuiseFramework API with guise 

### ISSUE THREAD

**Issue by ollieatkinson** [ollieatkinson/Guise#24]
We should generate the public API for guise so that we can review it

**Comment by Sherlouk** [comment]
How meta

**Comment by ollieatkinson** [comment]
We need to figure out what the build settings are when building the app with `swift build`, currently `guise` uses xcodebuild environment variables

**Comment by fortmarek** [comment]
Have you had time to look at this? I'd love to use this for my own tooling and I think it'd be a great addition!

**Comment by ollieatkinson** [comment]
If you're using Xcode it is super easy to add support, but I did not look into swift packages. I know there is a new swiftinterface file to go along with Swift5 so it might be worth looking at that if we can’t figure out a way to do it - I am pretty sure its just fiddling and finding out the right build settings that a package spits out

**Comment by fortmarek** [comment]
Yeah, I wanted it for Swift packages; I'll see if I have the time, thanks for the intro to this issue :)
